src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -292,6 +292,11 @@ public function loadByParent(
         return $children;
     }
 
+    /**
+     * @param string $uuid Path or UUID
+     * @param string|null $locale Locale
```

Comment:
locale should stay required here

Reply:
would keep the locale here required and not open the interface also when not handle currently in all cases correctly:

```suggestion
     * @param string $locale
```